contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -504,6 +505,11 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     bool nativePayment,
     bool onlyPremium
   ) internal view returns (uint96) {
+    // fulfilRandomWords msg.data has 772 bytes and with an additional
```

Comment:
How is this 772 bytes derived?

Reply:
+1 for this question, also maybe putting 804 in a constant within the contract

Reply:
@jinhoonbang and I ran VRF v2.5 requests with onlyPremium=True and onlyPremium=False for native payments and got the calldata msg.data length

Reply:
Perhaps we can add this to the comment? please verify @kidambisrinivas 

function signature - 4
pk - 64
gamma - 64
c - 32
s - 32
seed - 32
uWitness - 32
cGammaWitness - 64
sHashWitness - 64
zInv - 32
location of requestCommitment - 32
onlyPremium - 32
blockNum - 32
subId - 32
callbackGasLimit - 32
numWords - 32
sender - 32
extraArgs - 128

Reply:
Sure, Thanks!